Essentials/src/com/earth2me/essentials/commands/Commandtpaall.java

```diff
@@ -42,6 +45,11 @@ private void teleportAAllPlayers(final Server server, final CommandSource sender
                 continue;
             }
             try {
+            	TPARequestEvent tpaEvent = new TPARequestEvent(sender, player, true);
+                ess.getServer().getPluginManager().callEvent(tpaEvent);
```

Comment:
line spacing..

Reply:
It was caused by tab indentation, will fix in the next commit.